layers/+email/notmuch/packages.el

```diff
@@ -155,9 +155,10 @@
       (kbd "gr") 'notmuch-refresh-this-buffer
       (kbd "gR") 'notmuch-refresh-all-buffers
       (kbd "G") 'notmuch-search-last-thread
-      (kbd "M") 'compose-mail-other-frame)
+      (kbd "M") 'compose-mail-other-frame)))
 
-    (spacemacs|add-company-backends :backends notmuch-company :modes notmuch-message-mode)))
+(defun notmuch/post-init-company ()
```

Comment:
I think you still need to add company to the list of packages to have the post init event fire.

Reply:
Ah, yep, thanks for catching that.